src/ImuSensor_TEST.cc

```diff
@@ -191,19 +195,14 @@ TEST(ImuSensor_TEST, CreateImuSensor)
   sdf::ElementPtr imuSDF = ImuSensorToSDF(name, update_rate, topic,
     accelNoise, gyroNoise, always_on, visualize);
 
-  // Create an ImuSensor
-  auto sensor = mgr.CreateSensor<ignition::sensors::ImuSensor>(imuSDF);
```

Comment:
We shouldn't remove the `CreateSensor` tests, that function is still supported, right?

Instead, we should add separate tests for the `Load` functionality that you're using below.